Create registry_set_scr_file_executed_by_rundll32.yml

jstnk9 opened the pull request:
First PR. Let me know if I've to change anything please :)

frack113 commented:
Hi, 
when use graphical interface image is `rundll32.exe`
![image](https://user-images.githubusercontent.com/62423083/166718096-db68b9c2-de58-498f-aa11-12f6e5a805c0.png)

So will trigger when user change the screensaver

jstnk9 commented:
@frack113 thanks for the observation!

What about if we add a filter to excude Details from System32 and SysWOW64? Checking telemetry from a large number of endpoints, I can see that a very high percentage, if not almost all, of the value comes from those paths (system32 and syswow64).

Next image is just an example on my lab

**Excluding system32:**
![image](https://user-images.githubusercontent.com/7794663/167093386-34c2ed96-e5ec-4732-ba2f-684f4a291889.png)

query:

```
(winlog.event_data.EventType:"SetValue" AND winlog.event_data.Image:"*\\rundll32.exe") AND (winlog.event_data.TargetObject:"*\\Control Panel\\Desktop\\SCRNSAVE.EXE*" AND winlog.event_data.Details:*.scr) AND NOT (winlog.event_data.Details:"C:\\Windows\\System32\\*")
```

**Without excluding**
![image](https://user-images.githubusercontent.com/7794663/167093590-3364d49c-ad6b-4fdb-b2da-c8ccdf0b7e88.png)

query:

```
(winlog.event_data.EventType:"SetValue" AND winlog.event_data.Image:"*\\rundll32.exe") AND (winlog.event_data.TargetObject:"*\\Control Panel\\Desktop\\SCRNSAVE.EXE*" AND winlog.event_data.Details:*.scr)
```

jstnk9 commented:
new changes:

- deleted ScreenSaveActive registry key
- deleted ScreenSaveTimeOut key
- added filter to check if system32 or syswow64 on the registry key

frack113 commented on rules/windows/registry/registry_set/registry_set_scr_file_executed_by_rundll32.yml:
```diff
@@ -0,0 +1,34 @@
+        - 'C:\Windows\System32\'
+        - 'C:\Windows\SysWOW64\'
+        - 'C:\Windows\system32\'
+        - 'C:\Windows\SysWow64\'
```
Hi,
shoud be only `C:\Windows\System32` and `C:\Windows\SysWOW64`.
Elk use for text `keyword` type casa-sensitive or `text` type case-insensitive.
You have to use some sigmac option to deal with.
I kown elastic is a pain in the neck.

jstnk9 commented:
New commit with 2 paths only :)

frack113 reviewed (approved):
No FP with aurora tests
